dist/index.js

```diff
@@ -0,0 +1,2 @@
QUFjLFFBdUJuQ0MsWUFOa0IsV0FmUyxJQUNqQkMsRUFJQUMsRUFKQUQsRUFBZUgsU0FBU0MsY0FBYywyQkFDNUNGLEVBQUlNLFVBQVlGLEVBQWFFLFVBR3ZCRCxFQUFZSixTQUFTQyxjQUFjLHdCQUN6Q0YsRUFBSU0sV0FBYUQsRUFBVUMsVUFFRCxXQUMxQixJQUFNQyxFQUFlTixTQUFTQyxjQUFjLHVCQUN0Q0csRUFBWUosU0FBU0MsY0FBYyxzQkFDekNLLEVBQWFDLGlCQUFpQixTQUFTLFlEZlgsU0FBQ0MsRUFBaUNDLEdBRTdERCxFQUFnQkUsTUFBTUMsUUNjUyxPRFpoQyxJQUFNQyxFQUFhSixFQUFnQlAsY0FBYyxlQUNqRCxHQUFJVyxFQUFZLENBQ1gsSUFBTSxFQUFvQixXQUV0QkosRUFBZ0JFLE1BQU1DLFFBQVUsT0FFaENDLFdBQVlDLG9CQUFvQixRQUFTLElBR2pERCxXQUFZTCxpQkFBaUIsUUFBUyxJQ0duQ08sQ0FBaUJWLE1BTXJCVyxLQUdrQixNIiwiZmlsZSI6ImluZGV4LmpzIiwic291cmNlc0NvbnRlbnQiOlsiZXhwb3J0IGNvbnN0IHNob3dBUG9wVXBTY3JlZW4gPSAoc2NyZWVuVG9EaXNwbGF5OiBIVE1MRGl2RWxlbWVudCwgc3RhcnRpbmdQb3NpdGlvbjogc3RyaW5nKSA9PiB7XHJcbiAgICAvLyBkaXNwbGF5IHRoZSBoZWxwIHNjcmVlblxyXG4gICAgIHNjcmVlblRvRGlzcGxheS5zdHlsZS5kaXNwbGF5ID0gc3RhcnRpbmdQb3NpdGlvbjtcclxuICAgIC8vIGZ1bmN0aW9uIHdoaWNoIGdldHMgY2FsbGVkIHdoZW4gdGhlIGV4aXQgYnV0dG9uIGlzIGNsaWNrZWRcclxuICAgIGNvbnN0IGV4aXRCdXR0b24gPSBzY3JlZW5Ub0Rpc3BsYXkucXVlcnlTZWxlY3RvcignLmV4aXRCdXR0b24nKTtcclxuICAgIGlmIChleGl0QnV0dG9uKSB7XHJcbiAgICAgICAgIGNvbnN0IGV4aXRQb3BVcEZ1bmN0aW9uID0gKCkgPT4ge1xyXG4gICAgICAgICAgICAgLy9oaWRlIHRoZSBzY3JlZW5cclxuICAgICAgICAgICAgIHNjcmVlblRvRGlzcGxheS5zdHlsZS5kaXNwbGF5ID0gJ25vbmUnO1xyXG4gICAgICAgICAgICAgLy8gcmVtb3ZlIHRoZSBldmVudCBsaXN0ZW5lclxyXG4gICAgICAgICAgICAgZXhpdEJ1dHRvbj8ucmVtb3ZlRXZlbnRMaXN0ZW5lcignY2xpY2snLCBleGl0UG9wVXBGdW5jdGlvbik7XHJcbiAgICAgfTtcclxuICAgICAvLyBhZGQgZXZlbnQgbGlzdGVuZXIgdG8gdGhlIGV4aXQgYnV0dG9uO1xyXG4gICAgIGV4aXRCdXR0b24/LmFkZEV2ZW50TGlzdGVuZXIoJ2NsaWNrJywgZXhpdFBvcFVwRnVuY3Rpb24pO1xyXG4gICAgIH1cclxuICB9OyIsImltcG9ydCB7IHNob3dBUG9wVXBTY3JlZW4gfSBmcm9tICcuL3Nob3dBUG9wVXBTY3JlZW4nO1xyXG5cclxuY29uc3QgYXBwID0gZG9jdW1lbnQucXVlcnlTZWxlY3RvcignI2FwcCcpIGFzIEhUTUxEaXZFbGVtZW50XHJcblxyXG5jb25zdCByZW5kZXJTdGFydGluZ1BhZ2UgPSAoKSA9PiB7XHJcbiAgICBjb25zdCBzdGFydGluZ1BhZ2UgPSBkb2N1bWVudC5xdWVyeVNlbGVjdG9yKCcjc3RhcnRpbmctcGFnZS10ZW1wbGF0ZScpIGFzIEhUTUxUZW1wbGF0ZUVsZW1lbnQ7XHJcbiAgICBhcHAuaW5uZXJIVE1MID0gc3RhcnRpbmdQYWdlLmlubmVySFRNTDtcclxufVxyXG5jb25zdCByZW5kZXJJbmZvTW9kYWwgPSAoKSA9PiB7XHJcbiAgICBjb25zdCBpbmZvTW9kYWwgPSBkb2N1bWVudC5xdWVyeVNlbGVjdG9yKCcjaW5mby1tb2RhbC10ZW1wbGF0ZScpIGFzIEhUTUxUZW1wbGF0ZUVsZW1lbnQ7XHJcbiAgICBhcHAuaW5uZXJIVE1MICs9IGluZm9Nb2RhbC5pbm5lckhUTUw7XHJcbn1cclxuY29uc3QgY3JlYXRlSG93VG9QbGF5QnV0dG9uID0gKCkgPT4ge1xyXG4gICAgY29uc3QgaG93VG9QbGF5QnRuID0gZG9jdW1lbnQucXVlcnlTZWxlY3RvcignI2hvdy10by1wbGF5LWJ1dHRvbicpIGFzIEhUTUxVTGlzdEVsZW1lbnQ7XHJcbiAgICBjb25zdCBpbmZvTW9kYWwgPSBkb2N1bWVudC5xdWVyeVNlbGVjdG9yKCcjaW5mby1tb2RhbC1zY3JlZW4nKSBhcyBIVE1MRGl2RWxlbWVudDtcclxuICAgIGhvd1RvUGxheUJ0bi5hZGRFdmVudExpc3RlbmVyKCdjbGljaycsICgpID0+IHtcclxuICAgICAgICBzaG93QVBvcFVwU2NyZWVuKGluZm9Nb2RhbCwgJ2ZsZXgnKTtcclxuICAgIH0pXHJcbn1cclxuY29uc3QgcmVuZGVyQXBwID0gKCkgPT4ge1xyXG4gICAgcmVuZGVyU3RhcnRpbmdQYWdlKCk7XHJcbiAgICByZW5kZXJJbmZvTW9kYWwoKVxyXG4gICAgY3JlYXRlSG93VG9QbGF5QnV0dG9uKCk7XHJcbn1cclxuXHJcbnNldFRpbWVvdXQocmVuZGVyQXBwLCAyMDAwKTtcclxuIl0sInNvdXJjZVJvb3QiOiIifQ==
```

Comment:
we shouldn't track in git files from `dist` folder. See how to add it to .gitignore and how to remove it from tracked files:

https://stackoverflow.com/questions/59561657/how-can-i-ignore-dist-folder-in-gitignore-file